assets/css/components/post-block.scss

```
@@ -0,0 +1,136 @@
+.post-block {
+  position: relative;
+  display: grid;
+  grid-template-columns: 70% 30%;
```

```suggestion
  grid-template-columns: 2fr 1fr;
```

Where possible, when using CSS grid use the `fr` unit. It takes into account the grid gap when assigning space.